src/Catty/PlayerEngine/Sensors/SensorManager.swift

```diff
@@ -27,17 +27,23 @@ import CoreMotion
 
     public static var defaultValueForUndefinedSensor: Double = 0
     private static var sensorMap = [String: Sensor]() // TODO: make instance let
+    var landscapeMode: Bool = false
 
-    public required init(sensors: [Sensor]) {
+    public required init(sensors: [Sensor], landscapeMode: Bool) {
         super.init()
         registerSensors(sensorList: sensors)
+        setLandscapeMode(landscapeMode: landscapeMode)
     }
 
     private func registerSensors(sensorList: [Sensor]) {
         type(of: self).sensorMap.removeAll()
         sensorList.forEach { type(of: self).sensorMap[$0.tag()] = $0 }
     }
 
+    private func setLandscapeMode(landscapeMode: Bool) {
```

Comment:
Please set it directly in the initializer (will be necessary when changing to `let`).

Reply:
Done!